src/guide/9-visualisation/1-building-with-vis.rst

```diff
@@ -1,2 +1,11 @@
 Building with Visualisation
 ===========================
+
+To use the visualisation features you must ensure that you have followed the instructions for Building
+with visualisation enabled. If you wish to use your own visualiser, you can save the simulation state
+to disk as described in **TODO: link**. 
```

Comment:
Check for `VISUALISATION` macro being defined in C, or `pyflamegpu.VISUALISATION!=0` in py, this should be visible in examples.

Reply:
Though maybe this is the wrong place, more checking if built with vis support.